components/eamxx/cmake/machine-files/ascent.cmake

```diff
@@ -1,13 +1,15 @@
+include(${CMAKE_CURRENT_LIST_DIR}/common.cmake)
+common_setup()
+
 # Copied from summit.cmake, only change is to SCREAM_MACHINE
 # and remove support for CPU builds.
 #
 # Load all kokkos settings from Ekat's mach file
-set (EKAT_MACH_FILES_PATH ${CMAKE_CURRENT_LIST_DIR}/../../../../externals/ekat/cmake/machine-files)
 include (${EKAT_MACH_FILES_PATH}/kokkos/nvidia-v100.cmake)
 include (${EKAT_MACH_FILES_PATH}/kokkos/cuda.cmake)
+include (${EKAT_MACH_FILES_PATH}/mpi/other.cmake)
```

Comment:
I would add a comment here, specifying that this is sets all defaults to empty strings.